2021-01-14 11:08:36,914 - HMI.py[line:156] - INFO: {'1620M013AZ01': {'position': '-', 'status': '未确认', 'id': 0}, '1620M0298Z01': {'position': '-', 'status': '未确认', 'id': 1}, '1619M12285Z17': {'position': '-', 'status': '未确认', 'id': 2}, '1619M12287Z07': {'position': '-', 'status': '未确认', 'id': 3}}
2021-01-14 11:08:53,818 - HMI.py[line:156] - INFO: {'1620M013AZ01': {'position': '-', 'status': '未确认', 'id': 0}, '1620M0298Z01': {'position': '-', 'status': '未确认', 'id': 1}, '1619M12285Z17': {'position': '-', 'status': '未确认', 'id': 2}, '1619M12287Z07': {'position': '-', 'status': '未确认', 'id': 3}, 'status': '完好'}
2021-01-14 11:11:22,756 - HMI.py[line:156] - INFO: i-shopOCR user interface has been closed.

2021-01-14 11:11:37,616 - HMI.py[line:156] - INFO: {'1620M013AZ01': {'position': '-', 'status': '未确认', 'id': 0}, '1620M0298Z01': '完好', '1619M12285Z17': {'position': '-', 'status': '未确认', 'id': 2}, '1619M12287Z07': {'position': '-', 'status': '未确认', 'id': 3}}
2021-01-14 11:11:50,144 - HMI.py[line:156] - INFO: i-shopOCR user interface has been closed.

2021-01-14 11:14:50,297 - HMI.py[line:155] - INFO: i-shopOCR user interface has been closed.

2021-01-14 12:54:45,050 - HMI.py[line:156] - INFO: i-shopOCR user interface has been closed.

2021-01-14 13:36:28,239 - HMI.py[line:156] - INFO: i-shopOCR user interface has been closed.

2021-01-14 14:35:29,085 - HMI.py[line:234] - INFO: i-shopOCR user interface has been closed.

2021-01-14 14:36:24,401 - HMI.py[line:234] - INFO: i-shopOCR user interface has been closed.

2021-01-14 14:49:05,290 - HMI.py[line:252] - INFO: i-shopOCR user interface has been closed.

2021-01-14 14:56:40,156 - HMI.py[line:252] - INFO: Found camera NR0190090349, connecting...
2021-01-14 14:56:40,920 - HMI.py[line:252] - INFO: Successfully connected camera NR0190090349.
2021-01-14 14:57:14,624 - HMI.py[line:252] - INFO: Refreshing the camera configurations, restarting...
2021-01-14 14:57:14,850 - HMI.py[line:252] - INFO: Found camera NR0190090349, connecting...
2021-01-14 14:57:14,850 - HMI.py[line:252] - ERROR: Cannot connect the camera, please check the configurations and retry.
2021-01-14 14:59:57,052 - HMI.py[line:252] - INFO: Refreshing the camera configurations, restarting...
2021-01-14 14:59:57,302 - HMI.py[line:252] - INFO: Found camera NR0190090349, connecting...
2021-01-14 14:59:57,413 - HMI.py[line:252] - INFO: Successfully connected camera NR0190090349.
